ibm_catalog.json

```diff
@@ -545,14 +569,22 @@
             {
               "key": "existing_resource_group_name",
               "display_name": "resource_group",
+              "description": "Select or provide an existing IBM Cloud resource group. This value cannot be empty.",
               "custom_config": {
                 "type": "resource_group",
                 "grouping": "deployment",
                 "original_grouping": "deployment",
                 "config_constraints": {
                   "identifier": "rg_name"
                 }
-              }
+              },
+              "value_constraints": [
+                {
+                  "type": "regex",
+                  "description": "Resource group name cannot be empty.",
+                  "value": "^(?!\\s*$).+"
```

Comment:
This is pointless because we are using the catalog picker for resource groups so user will only be able to pick a resource group value from the dropdown